.fixtures.yml

```diff
@@ -2,7 +2,9 @@ fixtures:
   repositories:
     stdlib: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
     inifile: "https://github.com/puppetlabs/puppetlabs-inifile.git"
-    archive: "https://github.com/puppet-community/puppet-archive.git"
+    archive:
+      repo: "git://github.com/voxpupuli/puppet-archive.git"
```

Comment:
I roll-back the pinning, add some corrections in rspec file and now it's OK with the last archive version too !